3rdparty/openblas/openblas.cmake

```diff
@@ -18,12 +18,10 @@ ExternalProject_Add(
     URL https://github.com/xianyi/OpenBLAS/archive/refs/tags/v0.3.18.tar.gz
     URL_HASH SHA256=1632c1e8cca62d8bed064b37747e331a1796fc46f688626337362bf0d16aeadb
```

Comment:
[nit] We can update to v0.3.19. Looks like there are some bug fixes and support for newer Intel CPUs.